protostar/cli/common_arguments.py

```diff
@@ -0,0 +1,8 @@
+from .protostar_argument import ProtostarArgument
+
+lib_path_arg = ProtostarArgument(
+    name="lib-path",
+    description="Directory containing project dependencies. "
+    "This argument is/will after migrating the configuration file.",
```

Comment:
This seems a bit wrong

Reply:
[update description](https://github.com/software-mansion/protostar/pull/1058/commits/9e7079d021f1ed070be5939059f2b83f78125d86)